client/src/components/FooterComponent/FooterComponent.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const FooterComponent = ({ message }) => {
+  return <section className="Footer_component">{message}</section>;
```

Comment:
Footer_component : NO
Footer_container: YES

Reply:
Done

Reply:
regarding if we need a footer, which is not in the design mockup you shared, I included a footer because, that is generally a good place to link back to the repo. and more importantly, 

 - We absolutely need to _credit_ and _attribute_ the sources from where we got our questions, and also the attributions for any images we may require. 

Hence, a footer is required to include these considerations.

Reply:
`<footer>` tag for footer.